vidarr-pluginapi/src/main/java/ca/on/oicr/gsi/vidarr/api/SubmitWorkflowRequest.java

```diff
@@ -92,21 +92,20 @@ public String getWorkflowVersion() {
 
   @Override
   public int hashCode() {
-    return hashCodeIgnoreAttempt() * 31 + Integer.hashCode(attempt);
+    return hashCodeIgnoreAttemptAndConsumableResources() * 31 + Integer.hashCode(attempt);
   }
 
-  public int hashCodeIgnoreAttempt() {
+  public int hashCodeIgnoreAttemptAndConsumableResources() {
```

Comment:
Can a draft PR to shesmu be submitted to use this hashcode method?

Reply:
See above. I don't think having two hashcodes for the same object is the intention.